torch/export/__init__.py

```diff
@@ -717,7 +725,10 @@ def fn(x):
             else:
                 return x * 2
 
-    `export()` would give following error::
+    `v` is a data-dependent value, which is assumed to have a range of (-inf, inf).
+    :func:`export()` a hint about which branch to take would not be able to determine
+    if the traced branching decision is correct or not. Thus :func:`export()`
+    would give following error::
 
         torch._dynamo.exc.UserError: Consider annotating your code using
         torch.export.constrain_as_size() or torch.export().constrain_as_value() APIs.
```

Comment:
Github is dumb and won't let me comment on lines you haven't edited, but these lines should also be fixed: 

https://github.com/pytorch/pytorch/pull/108756/files#diff-5931bf805db4eb7fe55d85149992d6af82e0d7d7d29e2e0b69470329d6a0092dR739-R740
```
`v` --> ``v``
`constrain_as_value` --> :func:`constrain_as_value`
``` 

https://github.com/pytorch/pytorch/pull/108756/files#diff-5931bf805db4eb7fe55d85149992d6af82e0d7d7d29e2e0b69470329d6a0092dR753-R754
```
`export` --> :func:`export`
```
https://github.com/pytorch/pytorch/pull/108756/files#diff-5931bf805db4eb7fe55d85149992d6af82e0d7d7d29e2e0b69470329d6a0092dR1027-R1028
```
`%arg0_1` --> ``%arg0_1``
`%arg1_1` --> ``%arg1_1``
```
https://github.com/pytorch/pytorch/pull/108756/files#diff-5931bf805db4eb7fe55d85149992d6af82e0d7d7d29e2e0b69470329d6a0092dR1067
```
`if` --> ``if``
```